packages/trendmicro/data_stream/deep_security/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,103 @@
+      ignore_missing: true
+  - rename:
+      field: cef.name
+      target_field: trendmicro.event.name
+      ignore_missing: true
+  - convert:
+      field: trendmicro.event.signature_id
+      type: integer
+      ignore_missing: true
+      # ignore_failure: true
+
+  # pipeline per signature id
+  - pipeline:
+      name: '{{ IngestPipeline "application-control-event" }}'
+      if: "ctx.trendmicro?.event.signature_id >= 6000000 && ctx.trendmicro?.event.signature_id <= 6999999"
+  - pipeline:
+      name: '{{ IngestPipeline "firewall-event" }}'
+      if: "ctx.trendmicro?.event.signature_id == 20 || ctx.trendmicro?.event.signature_id == 21"
+  - pipeline:
+      name: '{{ IngestPipeline "intrusion-prevention-event" }}'
+      if: "ctx.trendmicro?.event.signature_id == 10 || (ctx.trendmicro?.event.signature_id >= 1000000 && ctx.trendmicro?.event.signature_id <= 1999999)"
+  - pipeline:
+      name: '{{ IngestPipeline "integrity-monitoring-event" }}'
+      if: "ctx.trendmicro?.event.signature_id == 30 || (ctx.trendmicro?.event.signature_id >= 2000000 && ctx.trendmicro?.event.signature_id <= 2999999)"
+  - pipeline:
+      name: '{{ IngestPipeline "malware-event" }}'
+      if: "ctx.trendmicro?.event.signature_id >= 4000000 && ctx.trendmicro?.event.signature_id <= 4999999"
+  - pipeline:
+      name: '{{ IngestPipeline "system-event" }}'
+      if: "ctx.trendmicro?.event.signature_id >= 100 && ctx.trendmicro?.event.signature_id <= 7499"
+  - pipeline:
+      name: '{{ IngestPipeline "log-inspection" }}'
+      if: "ctx.trendmicro?.event.signature_id == 40 || (ctx.trendmicro?.event.signature_id >= 3000000 && ctx.trendmicro?.event.signature_id <= 3999999)"
+  - pipeline:
+      name: '{{ IngestPipeline "web-reputation" }}'
+      if: "ctx.trendmicro?.event.signature_id >= 5000000 && ctx.trendmicro?.event.signature_id <= 5999999"
+
+  - convert:
+      field: file.size
+      type: long
+      ignore_missing: true
+      ignore_failure: true
```

```suggestion
```
Unless it's an edge case scenario, we always want the failure to occur, so that we can do RCA properly and not sit on silent errors.